angular/src/app/shared/data-table/data-table.component.ts

```diff
@@ -30,6 +30,8 @@ export class DataTableComponent {
   rowSelection: TableRowSelection = TableRowSelection.single;
   @Input()
   pagination = true;
+  @Input()
+  hideBorder = false;
```

Comment:
Why making an input? What would be the use-case? Should be `1px` **always**.

Reply:
The default behaviour is to have a border around the grid, not including the pagination. The use case I was think was now, when we want to hide the default border and the parent decides what border it wants around the whole component. But I guess you're right, that doesn't need to be customizable, we can just always override the default border. I'll update this.